modern/mdnetwork/proxy.go

```diff
@@ -0,0 +1,23 @@
+package mdnetwork
+
+import (
+	"net/http"
+
+	"github.com/short-d/app/fw"
+)
+
+var _ fw.Network = (*Proxy)(nil)
+
+type Proxy struct {
+}
+
+func (p Proxy) FromHTTP(request http.Request) fw.Connection {
```

Comment:
Wouldn't it be better if we pass the request variable as pointer?

Reply:
Or maybe just the request.Header?

Reply:
@arberiii Done.